src/var.py

```diff
@@ -99,17 +99,18 @@
 
 # Legacy APIs that use v1 discovery. Newer APIs should all use v2.
 V1_DISCOVERY_APIS = {
-  u'oauth2',
-  u'reseller',
-  u'storage',
-  u'drive',
   u'admin',
+  u'appsactivity',
+  u'calendar',
+  u'drive',
+  u'drive3',
```

Comment:
this shouldn't' be necessary as we should have already switched to the true API name / version by now. This is the case with "admin" instead of directory / reports / datatransfer. Are you seeing differently?